src/Razor/src/Microsoft.VisualStudio.RazorExtension/Microsoft.VisualStudio.RazorExtension.Custom.pkgdef

```diff
@@ -11,6 +11,12 @@
 [$RootKey$\TextMate\LanguageConfiguration\ContentTypeMapping]
 "RazorLSP"="$PackageFolder$\language-configuration.json"
 
+[$RootKey$\TextMate\LanguageConfiguration\GrammarMapping]
+"text.html.basic"="$PackageFolder$\html-language-configuration.json"
+"source.js"="$PackageFolder$\javascript-language-configuration.json"
+"source.css"="$PackageFolder$\css-language-configuration.json"
+"source.cs"="$PackageFolder$\csharp-language-configuration.json"
```

Comment:
@allisonchou OH, was the answer to your question from earlier "We're not actually doing that for VS but we should be"?

Reply:
Haha after talking to Taylor, it was actually that the core editor needs to fix a TextMate bug (the one mentioned in https://github.com/dotnet/aspnetcore/issues/26975) and then we can re-enable TextMate for C#.